sdk/trace/src/test/java/io/opentelemetry/sdk/trace/export/BatchSpanProcessorTest.java

```diff
@@ -412,16 +442,24 @@ public String getDescription() {
         SdkTracerProvider.builder()
             .addSpanProcessor(
                 BatchSpanProcessor.builder(waitingSpanExporter)
-                    .setScheduleDelayMillis(MAX_SCHEDULE_DELAY_MILLIS)
+                    .setScheduleDelay(MAX_SCHEDULE_DELAY_MILLIS, TimeUnit.MILLISECONDS)
                     .setExportOnlySampled(true)
                     .build())
             .setTraceConfig(traceConfig::get)
             .build();
 
-    createEndedSpan(SPAN_NAME_2);
+    createEndedSpan(SPAN_NAME_1);
+    traceConfig.set(traceConfig.get().toBuilder().setSampler(Sampler.alwaysOn()).build());
+    ReadableSpan span = createEndedSpan(SPAN_NAME_2);
 
+    // Spans are recorded and exported in the same order as they are ended, we test that a non
```

Comment:
@jkwatson I ended up adding this as discussed on the other PR so the test ends in 100ms instead of 10s

Reply:
doh. I merged that one already, so you'll need a rebase.